src/tests/functional/shared_test_classes/src/single_layer/memory.cpp

```diff
@@ -191,9 +192,9 @@ void MemoryTest::CreateCommonFunc() {
                                    : VariableInfo{inputShape, ngPrc, "v0"};
     auto variable = std::make_shared<Variable>(variable_info);
     auto read_value = CreateReadValueOp(param.at(0), variable);
-    auto add = std::make_shared<Add>(read_value, param.at(0));
+    auto add = std::make_shared<ov::op::v1::Add>(read_value, param.at(0));
     auto assign = CreateAssignOp(add, variable);
-    auto res = std::make_shared<Result>(add);
+    auto res = std::make_shared<ov::op::v0::Result>(add);
```

Comment:
Is it needed? `using`'s are declared at the top.

Reply:
No, but because single_layer/ folder will be removed, it doesn't matter